dav-controller/api/routers/acapy_handler.py

```diff
@@ -44,13 +46,41 @@ async def post_topic(request: Request, topic: str, db: Database = Depends(get_db
 
         if webhook_body["state"] == "presentation_received":
             logger.info("GOT A PRESENTATION, TIME TO VERIFY")
-            client.verify_presentation(auth_session.pres_exch_id)
+            # client.verify_presentation(auth_session.pres_exch_id)
```

Comment:
This is removed because we have the `AUTO_VERIFY_PRESENTATIONS` flag set to true, correct? Might as well delete it if it is not used.